app/src/main/java/org/mozilla/fenix/search/SearchFragment.kt

```diff
@@ -38,12 +39,13 @@ class SearchFragment : Fragment() {
         val sessionId = SearchFragmentArgs.fromBundle(arguments!!).sessionId
         val isPrivate = SearchFragmentArgs.fromBundle(arguments!!).isPrivateTab
         val view = inflater.inflate(R.layout.fragment_search, container, false)
+        val url = if (sessionId == null) "" else requireComponents.core.sessionManager.findSessionById(sessionId)?.url!!
```

Comment:
You can remove the if/else and doublebang if you write this like: 

`val url = requireComponents.core.sessionManager.findSessionById(sessionId)?.let { it.url } ?: ""`

Reply:
Updated this line to be a little cleaner 😄